sdk/resourcemanager/Azure.ResourceManager/src/ArmClient.cs

```diff
@@ -96,6 +96,11 @@ public ArmClient(TokenCredential credential, string defaultSubscriptionId, Uri b
                 new Subscription(this, Subscription.CreateResourceIdentifier(defaultSubscriptionId));
         }
 
+        private static string GetAuthenticationScope(ArmAudience audience)
```

Comment:
Should this be a method on ArmAudience itself?

Reply:
I add `ArmAudience.DefaultScope` property. I also add `ArmAudience.GetScope` method, but keep it private since I'm not sure if there is any use case that people want to compose other kinds of scope.